### PR TITLE
feat: categories list added to homepage

### LINKED ISSUE #21: Add categories on home page and refactore home page
- Add categories on home page
- Refactor home page
- fix background image size.

### PR DESCRIPTION
**Pull request by rdshinde**
- categories list added to homepage
- Toast position refactored
- price filter refactored
- updated imports

### CODE REVIEW

File: src/pages/home/Home.jsx [hunk 1]
```diff
@@ -1,7 +1,23 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import "./home.css";
+import { useNavigate } from "react-router-dom";
+import { useFilter } from "../../contexts/filter/FilterContext";
```
Comment (rahulcs754) [inline src/pages/home/Home.jsx]: you can create index.js file in context and export all context in that file. so you have to not write long path here

File: src/pages/home/Home.jsx [hunk 2]
```diff
@@ -1,7 +1,23 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import "./home.css";
+import { useNavigate } from "react-router-dom";
```
Comment (rahulcs754) [inline src/pages/home/Home.jsx]: Link and useNavigat come in one line

**Review by rahulcs754 — APPROVED** [review]
The code look nice and clean 
just small suggestion that will help in code

File: src/pages/home/Home.jsx (hunk 1 above)
Comment (rdshinde) [reply src/pages/home/Home.jsx]: Sure. I will do that

File: src/pages/home/Home.jsx (hunk 2 above)
Comment (rdshinde) [reply src/pages/home/Home.jsx]: Auto import 🥲 .
I will fix that.

**Review by raman1999 — APPROVED** [review]
all looks good,  good naming conventions done

**Pull request merged**